easybuild/easyconfigs/p/PCL/PCL-1.7.2-intel-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'PCL'
+version = '1.7.2'
+versionsuffix = '-Python-2.7.11'
+
+homepage = 'http://pointclouds.org/'
+description = """The Point Cloud Library (PCL) is a standalone, large scale, open project for 2D/3D image and
+ point cloud processing."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/PointCloudLibrary/pcl/archive/']
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [('CMake', '3.5.2')]
+dependencies = [
+    ('Boost', '1.60.0', versionsuffix),
+    ('Eigen', '3.2.8'),
+    ('FLANN', '1.8.4', versionsuffix),
+    ('VTK', '6.3.0', versionsuffix),
+    ('Qhull', '2015.2'),
+]
+
+configopts = '-DCMAKE_BUILD_TYPE=RELEASE '
```

Comment:
It should be case sensitive I think. Not sure. Check if they do anthing special in their `CMakeLists.txt`?